sys/include/net/ng_netconf.h

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (C) 2015 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    net_netapi Configuration options for network APIs
+ * @ingroup     net
+ * @brief       List of available configuration options for network APIs
+ * @{
+ *
+ * @file
+ * @brief       Definition of global configuration options for netdev and netapi
+ *
+ * @author      Hauke Petersen <hauke.petersen@fu-berlin.de>
+ */
+
+#ifndef NET_CONF_H_
+#define NET_CONF_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   Global list of configuration options available throughout the
+ *          network stack, e.g. by netdev and netapi
```

Comment:
nope, they are also intended to be netapi options. For one they make sense to be used in netapi when send to the mac layer (just think of the `addr` shell command). Also you can set addresses for interfaces in the IP layer and so on. Sure some options are link layer specific, but you never implement all of them...